src/microbe_stage/MicrobeStage.cs

```diff
@@ -507,9 +507,17 @@ public override void OnReturnFromEditor()
             wonOnce = true;
         }
 
+        // Update the player's cell
+        Player!.ApplySpecies(Player.Species);
+
+        // Reset all the duplicates organelles of the player
+        Player.ResetOrganelleLayout();
+
+        var playerPosition = Player.GlobalTransform.origin;
+
         // Spawn another cell from the player species
-        // This is done first to ensure that the player colony is still intact for spawn separation calculation
```

Comment:
This isn't a request to change anything, but just so that I understand what's going on, does this mean now that the player colony is already reset before dividing now? Or does `ResetOrganelleLayout` not reset the colony? Hopefully a bug where the player colony is never reset is not present now with these changes.

Reply:
It's not really intended. It's a known problem, here's the issue I opened about despawning not working: https://github.com/Revolutionary-Games/Thrive/issues/3676 I'll comment there that resetting the organelle layout should also be made to properly despawn the colony members.